docker-app/qfieldcloud/filestorage/view_helpers.py

```diff
@@ -233,6 +238,36 @@ def download_field_file(
     http_host = request.headers.get("host", "")
     https_port = http_host.split(":")[-1] if ":" in http_host else "443"
 
+    download_range = request.headers.get("Range", "")
+    if download_range:
+        range_match = parse_range(download_range)
+
+        if not range_match:
+            return HttpResponse(
+                "`Range` HTTP header must be formatted like `bytes=start-(end)`",
+                status=416,
+            )
+
+        file_size = field_file.size
+        range_start = range_match[0]
+
+        if range_match[1]:
+            range_end = range_match[1]
+        else:
+            range_end = file_size - 1
+
+        if (
+            range_start >= file_size
+            or range_end >= file_size
+            or range_end < range_start
+        ):
+            return HttpResponse(
+                "Invalid value for `Range` HTTP header`",
+                status=416,
+            )
+
+        range_length = range_end - range_start + 1
```

Comment:
```suggestion
        range_length = range_end - range_start + 1
        
        if range_length < settings.QFIELDCLOUD_MINIMUM_RANGE_HEADER_LENGTH:
            raise InvalidRangeError("Requested range too small, expected at least {} but got {} bytes".format(settings.QFIELDCLOUD_MINIMUM_RANGE_HEADER_LENGTH, range_length))
```

A few suggestions in this comment:
1) We should add a minimum range of 1MB or so, otherwise someone can DDOS us without even realizing.
2) note the `InvalidRangeError`, which can be added to `qfieldcloud.core.exceptions` instead of manually returning `HttpResponse` here and there. Then the default handler as defined in `rest_utils` should convert it to the proper error.
3) Please do not hardcode the status to 416, use the `rest_framework.status.HTTP_416_REQUESTED_RANGE_NOT_SATISFIABLE` instead of the raw value.